ui/components/MesheryPerformance/PerformanceCard.js

```diff
@@ -94,30 +197,38 @@ function PerformanceCard({
       {/* FRONT PART */}
       <>
         <div>
-          <Typography variant="h6" component="div">
-            {name}
-          </Typography>
+          <div style={{ display : "flex", justifyContent : "space-between" }}>
+            <Typography variant="h6" component="div">
+              {name}
+            </Typography>
+            <img src={`/static/img/load-test/${loadGenerators[0]}.svg`} height="24px" />
+          </div>
           <div className={classes.noOfResultsContainer} >
             <div >
```

Comment:
Not sure why we have gap in `<div > and <div >`  in line number 206 and 207 respectively
Can we remove it?